lib/backends/secrets-manager-backend.js

```diff
@@ -39,6 +39,9 @@ class SecretsManagerBackend extends KVBackend {
       .getSecretValue({ SecretId: secretKey })
       .promise()
 
+    if (data.SecretString === undefined) {
```

Comment:
can you tweak this to explicitly check for the existence of the properties?  I.e.,

```js
if ('SecretBinary' in data) {
  return data.SecretBinary
} else if ('SecretString' in data) {
  return data.SecretString
}

this._logger('Unexpected data from Secrets Manager secret ${secretKey}')
return null
```

Reply:
@silasbw thank you for the comment, I fixed it.